generators/profilegen/src/profile_generator.rb

```diff
@@ -126,7 +121,7 @@ def check_available_after_index(operation_type_to_check, index_after_to_compare)
   #
   def generate_client
     file = get_client_file
```

Comment:
similar to a comment I made in another place, should we have a method to create a file based on a name and template, seems that we do that in multiple places and not very consistent across the generator methods.